airflow/ui/package.json

```diff
@@ -18,10 +18,14 @@
   "dependencies": {
     "@chakra-ui/anatomy": "^2.2.2",
     "@chakra-ui/react": "^2.8.2",
+    "@codemirror/lang-json": "^6.0.1",
+    "@codemirror/theme-one-dark": "^6.0.0",
```

Comment:
In light mode the codemirror was always showing dark mode. THat disturbed a bit. I'd propose to ajust the coloring like the rest of the UI. Looks like this in light mode:

![image](https://github.com/user-attachments/assets/5a6642cf-4745-4631-ba64-e1923725bc30)

...compared to this in ark mode...

![image](https://github.com/user-attachments/assets/97763d6d-5dde-44c7-aaa0-88e482841d89)

Reply:
Yaa, This is TODO, will try something dynamic.